charts/images.yaml

```diff
@@ -190,7 +190,7 @@ images:
 # Miscellaenous
 - name: alpine
   repository: alpine
-  tag: "3.10.3"
+  tag: "3.13.5"
```

Comment:
I guess it is better to update this with `alpine-iptables` together. @DockToFuture , can you please push `3.13.5` tag for the alpine-iptables image?

Reply:
It is a bit unfortunate that the repository https://github.com/DockToFuture/alpine-iptables no longer exist.
And we had discussion about this image long time ago here https://github.com/gardener/gardener/pull/1644.

Reply:
FYI: There is #4574 now.